src/gui_common/dialogs/CustomConfirmationDialog.cs

```diff
@@ -139,7 +143,17 @@ private void UpdateLabel()
         if (dialogLabel == null)
             throw new SceneTreeAttachRequired();
 
-        dialogLabel.Text = TranslationServer.Translate(dialogText);
+        var text = new StringBuilder(100);
+
+        if (CenterText)
```

Comment:
I think for non-centered text it would be very nice to have a higher performance variant where the StringBuilder is not used at all.

Reply:
Done.